index.js

```diff
@@ -136,6 +136,21 @@ function serialize(name, val, options) {
     str += '; Path=' + opt.path;
   }
 
+  if (null != opt.sameSite) {
+    var sameSiteRegExp = new RegExp(/^(strict|lax)$/i);
```

Comment:
`new RegExp` is not necessary, you can just use a literal. Also, please hoist the expression up with the other expressions, since v8 is very slow when compiling expressions.